File: java/Insert.java
import java.sql.*;

public class Insert {
    public static void main(String[] args) {
        String database = "project280";
        String user = "user280";
        String password = "p4ssw0rd";

        String url = "jdbc:mysql://localhost:3306/" + database;

        try {
            Connection con = DriverManager.getConnection(url, user, password);
            Statement st = con.createStatement();

            if (args.length != 4) {
                System.err.println("error: invalid arguments\nusage: <first name> <last name> <phone> <age>");
                System.exit(1);
            }

            PreparedStatement pst = con.prepareStatement("INSERT INTO people (first_name, last_name, phone, age, time_added) VALUES (?,?,?,?,NOW())");

            for (int i=1; i <= 4; i++)
                pst.setString(i, args[i - 1]);

            pst.executeUpdate();

        } catch (SQLException ex) {
            System.err.println("SQL error: " + ex.getMessage());
        }
    }
}

File: php/insert.php
<?php
$HOST = '127.0.0.1';
$USER = 'user280';
$PASS = 'p4ssw0rd';
$DB = 'project280';
ini_set('display_errors', 1);
ini_set('display_startup_errors', 1);
error_reporting(E_ALL);

$dsn = "mysql:host=$HOST;dbname=$DB";
$pdo = new PDO($dsn, $USER, $PASS);

if ($_POST['age'] > 80) {
	echo "Too old!";
	die();
}

$stmt = $pdo->prepare('INSERT INTO People (first_name, last_name, phone, age, time_added) VALUES (?, ?, ?, ?, NOW())');
$stmt->execute([$_POST['firstname'], $_POST['lastname'], $_POST['phone'], $_POST['age']]);

if ($pdo->lastInsertId() == "0") {
  setcookie("ShowError", "True");
}
else {
  setcookie("ShowSuccess", "True");
}

header('Location: /index.php') ;
?>
